packages/connect-web/src/utils/proxy-event-emitter.ts

```diff
@@ -0,0 +1,89 @@
+import EventEmitter from 'events';
+
+export default class ProxyEventEmitter implements EventEmitter {
```

Comment:
maybe please add a comment why is this needed, I remember we were discussing this and it made sense back then but after 3 weeks I am completely lost :D